### PR TITLE
Update the two FP16 models

### PR DESCRIPTION
**Pull request by Charles-258**
Fixes https://github.com/opencv/opencv_zoo/issues/122

I update the two FP16 models, and the results of two models remain the same. Besides, I delete the INT8 models, which can keep the best models of CRNN.

### CODE REVIEW

File: models/text_recognition_crnn/README.md [hunk 1]
```diff
@@ -8,12 +8,10 @@ Results of accuracy evaluation with [tools/eval](../../tools/eval) at different
 | ------------ | ---------- | --------- | --------- |
 | CRNN_EN      | 81.66      | 74.33     | 52.78     |
 | CRNN_EN_FP16 | 82.01      | 74.93     | 52.34     |
-| CRNN_EN_INT8 | 81.75      | 75.33     | 52.43     |
 | CRNN_CH      | 71.28      | 80.90     | 67.36     |
 | CRNN_CH_FP16 | 78.63      | 80.93     | 67.01     |
-| CRNN_CH_INT8 | 78.03      | 80.93     | 66.67     |
 
-\*: 'FP16' or 'INT8' stands for 'model quantized into FP16' or 'model quantized into int8'
```
Comment (fengyuentau) [inline models/text_recognition_crnn/README.md:16]: Any context regarding the removal of int8 models?

Comment (Charles-258) [reply models/text_recognition_crnn/README.md:16]: We decided to retain int8 models after disscussion, and I will re-upload the int8 models.

Comment (zihaomu) [reply models/text_recognition_crnn/README.md:16]: I propose to keep the original int8 model, since the accuracy is the same.

**Review by zihaomu — APPROVED** [review]
👍

**Comment by zihaomu** [comment]
Hi @Charles-258, can you recheck if this part can be simplified?
Other part looks good to me.
image

File: models/text_recognition_crnn/README.md (hunk 1 above)
Comment (fengyuentau) [reply models/text_recognition_crnn/README.md:16]: > retain int8 models

Okay

**Comment by fengyuentau** [comment]
> can you recheck if this part can be simplified?

Hello @Charles-258 , do you plan to do this check?

**Comment by Charles-258** [comment]
> > can you recheck if this part can be simplified?
> 
> Hello @Charles-258 , do you plan to do this check?

Sorry for the late reply. I can simplify this part before tomorrow.

**Comment by zihaomu** [comment]
Hi @Charles-258, I uploaded the model for you. Please take a look.
The simplified credit is to @Charles-258.

**Review by zihaomu — APPROVED** [review]

File: models/text_recognition_crnn/README.md (hunk 1 above)
Comment (fengyuentau) [reply models/text_recognition_crnn/README.md:16]: Does the accuracy of EN_FP16 change since the model is updated?

**Review by fengyuentau — CHANGES_REQUESTED** [review]
Please remove extra files:
- .DS_Store
- models/.DS_Store
- models/text_recognition_crnn/.DS_Store

Also rename the new model files to remove _sim suffix.

**Comment by zihaomu** [comment]
Done

**Comment by zihaomu** [comment]
Done

File: models/text_recognition_crnn/README.md (hunk 1 above)
Comment (zihaomu) [reply models/text_recognition_crnn/README.md:16]: The accuracy is the same.

**Pull request merged**